t/python/t0005-kvs.py

```diff
@@ -334,6 +335,50 @@ def test_36_put_symlink(self):
         flux.kvs.commit(self.f)
         self.assertFalse(flux.kvs.exists(self.f, "txn_symlink"))
 
+    def bad_input(self, func, *args):
+        got_einval = False
+        try:
+            func(*args)
+        except EnvironmentError as e:
+            if e.errno == errno.EINVAL:
+                got_einval = True
+        self.assertEqual(got_einval, True)
+
+    def test_37_exists_bad_input(self):
+        self.bad_input(flux.kvs.exists, self.f, "")
+
+    def test_38_isdir_bad_input(self):
+        self.bad_input(flux.kvs.isdir, self.f, "")
+
+    def test_39_get_bad_input(self):
+        self.bad_input(flux.kvs.get, self.f, "")
+
+    def test_40_get_dir_bad_input(self):
+        self.bad_input(flux.kvs.get_dir, self.f, "")
+
+    def test_41_put_exception_bad_input(self):
+        self.bad_input(flux.kvs.put, self.f, "", "")
+
+    def test_42_put_mkdir_exception_bad_input(self):
+        self.bad_input(flux.kvs.put_mkdir, self.f, "")
+
+    def test_43_put_unlink_bad_input(self):
+        self.bad_input(flux.kvs.put_unlink, self.f, "")
+
+    def test_44_put_symlink_bad_input(self):
+        self.bad_input(flux.kvs.put_symlink, self.f, "", "")
+
+    # try to overwrite root dir, will fail on commit
+    def test_45_commit_fail(self):
+        got_einval = False
+        try:
+            flux.kvs.put(self.f, ".", "foof")
+            flux.kvs.commit(self.f)
+        except EnvironmentError as e:
+            if e.errno == errno.EINVAL:
+                got_einval = True
```

Comment:
Same as above, use `self.assertRaises()` with a context to check expected contents of the exception.